CHANGES.md

```diff
@@ -0,0 +1,22 @@
+# History of changes and releases
+
+Release dates, binaries. and other information for the Trino Python client are
+available with the [tags
+list](https://github.com/trinodb/trino-python-client/tags) and the
+[README](https://github.com/trinodb/trino-python-client/blob/master/README.md).
+
+## Trino Python client 0.316.0
+
+* Add support for SQLAlchemy queries to access multiple catalogs by specifying
+  a `trino_catalog` argument to SQLAlchemy `Table` objects. (#186)
```

Comment:
Does `#186` link to a PR or it misses a link?

Reply:
good point. It looks like the magic linking only works in PR/issue comments and commit messages.

I'll fix this.

Reply:
Looks like this now. Thanks for catching.
![image](https://user-images.githubusercontent.com/10473142/191476944-9bc45c6d-7950-4737-af9d-658d5b33f9d9.png)